packages-backend/express/src/auth.ts

```diff
@@ -36,47 +78,59 @@ const throwIfIssuerIsNotAValidUrl = (issuer: string) => {
     );
   }
 };
-
-type TDecodedJWT = {
-  sub: string;
-  iss: string;
-  [property: string]: string;
+// Validates required option values.
+const validateRequiredValues = (options: TSessionMiddlewareOptions) => {
```

Comment:
Looked into Node.js `assert` but somehow it wouldn't help with this. Was just curious.